src/i18n/en-events.ts

```diff
@@ -6,6 +6,7 @@ const eventsMessageDescriptors = {
   'event.title': `{eventName, select,
       ${ColonyAndExtensionsEvents.OneTxPaymentMade} {{initiator} paid {amount} {tokenSymbol} from {fromDomain} to {recipient}}
       ${ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots} {{initiator} transferred {amount} {tokenSymbol} from the {fromDomain} to the {toDomain}}
+      ${ColonyAndExtensionsEvents.TokensMinted} {{initiator} minted {amount} {tokenSymbol} to {recipient}}
```

Comment:
This should show the colony name (in place of the recipient) if one exists, the same thing we do for the user.

So you'll need to create a similar component to my `FriendlyUserName` _(check `ActionsPage` to see how that works)_